presto-main/src/test/java/io/prestosql/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -415,6 +415,28 @@ public static PlanMatchPattern join(
                 .with(joinMatcher);
     }
 
+    public static PlanMatchPattern join(
```

Comment:
I think this belong to previous commit

Reply:
The changes to use this function in `TestDynamicFilter` are part of this commit, that's why I've kept it in this commit.